plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1286,6 +1287,7 @@ blockly.block.wander=Wander around (enables walking) with speed factor %1 %2
 blockly.block.wander_avoid_water=Wander around and avoid water with speed factor %1 %2
 blockly.block.biome_entered=If player enters biome %1
 blockly.block.block_placed=If %1 is placed
+blockly.block.entity_hurt_player=If player gets hit by %3 with damage between %1 and %2 Shield active: %4
```

Comment:
`between %1 and %2 Shield active: %4`

we usually don't capitalize in middle of procedure block, so more fitting with other blocks would be

`between %1 and %2 shield active: %4`

or

`between %1 (%2 shield active: %4)`

See some similar blocks from AI section:

![image](https://github.com/MCreator/MCreator/assets/16374228/2ca77b51-1ec8-46dd-b122-2067711fb77c)

Reply:
i will remember that for next time, 
I removed the Capital